docs/product_search/parameters.md

```diff
@@ -0,0 +1,240 @@
+Will return experiences which have a [duration]({% link docs/reference/objects/product/index.md %}#productduration) within the defined range.<br>
+Note: Experience [durations]({% link docs/reference/objects/product/index.md %}#duration) can be returned as a number of hours, 1 day or a number of nights, whereas product_search `duration` always takes the duration as a number of days.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search duration: {greater_than: 3, less_than: 8 } %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as query paramameters
+```
+https://mysite.com/search?search[duration][greater_than]=3&search[duration][less_than]=8
+```
+
+### exclude_sold_out_products
+Accepts: `true` or `false`.<br>
+Passing `true` will exclude [sold out]({% link docs/reference/objects/product/index.md %}#productsold_out) products from the products returned.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search exclude_sold_out_products: true %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[exclude_sold_out_products]=true
+```
+
+### include_organisation_products
+Accepts: `true` or `false`.<br>
+Passing `true` will include products from other companies which are linked in the same organisation.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search include_organisation_products: true %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[include_organisation_products]=true
+```
+
+### name
+Executes a partial search on the string passed in.<br>
+Will return any products whose [name]({% link docs/reference/objects/product/index.md %}#productname) matches the argument.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search name: 'my experience' %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[name]=my+experience
+```
+
+### series_id
+Accepts a series id.<br>
+Will return products which match the [series id]({% link docs/reference/objects/series.md %}#seriesid) passed.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search series_id: 'abcd1234-1234-abcd-1234-abcd1234abcd' %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[series_id]=abcd1234-1234-abcd-1234-abcd1234abcd
+```
+
+### subcategory
+Accepts any of the Easol predefined subcategories.<br>
+Will accept either a single subcategory or an array.<br>
+Will return products which match the [subcategory]({% link docs/reference/objects/product/index.md %}#productsubcategory) passed.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search subcategory: 'wellness' %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[subcategory]=wellness
```

Comment:
Be helpful to see an example with array of subcategories as well

Reply:
I couldn't figure that one out, I suspect its the same as countries. I've added category and subcategory to the country array ticket as well so we can follow up with the platform team.